.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "*.yaml": "home-assistant"
+    }
+}
```

Comment:
Please don't add `.vscode` settings to the project. Add them to your `.gitignore` or project's `.gitignore`

Reply:
Done